topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md

```diff
@@ -0,0 +1,759 @@
+
+This is our own formulation of the barcode plot based on a [discussion](https://github.com/COMBINE-lab/salmon/issues/362#issuecomment-490160480) we had with community members. The left hand plots with the smooth lines are the main plots, showing the UMI counts for individual cell barcodes ranked from high to low. We expect a sharp drop-off between cell-containing droplets and ones that are empty or contain only cell debris. Now, this data is not an ideal dataset, so for perspective, in an ideal world with a very clean 10x run, data will look a bit more like the following (see the study in Single Cell Expression Atlas [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6653/results/tsne) and the project submission [here](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6653/)).
+
+![raw droplet barcode plots - lung atlas](../../images/wab-lung-atlas-barcodes-raw.png "Pretty data - raw")
+
+In that plot, you can see the clearer 'knee' bend, showing the cut-off between empty droplets and cell-containing droplets.
+
+The right hand plots are density plots from the first one, and the thresholds are generated either using [dropletUtils](https://bioconductor.org/packages/release/bioc/html/DropletUtils.html) or by the method described in that discussion. We could use any of these thresholds to select cells, assuming that anything with fewer counts is not a valid cell. By default, Alevin does something similar, and we can learn something about that by plotting just the barcodes Alevin retains.
+
+> ### {% icon hands_on %} Hands-on: Generate Alevin's barcode QC plot
+>
+> 1. {% tool [Droplet barcode rank plot](https://humancellatlas.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/droplet_barcode_plot/_dropletBarcodePlot/1.6.1+galaxy2) %} with the following parameters:
+>    - *"Input MTX-format matrix?"*: `Yes`
+>    - *"Matrix-market format matrix file, with cells by column (overrides --barcode-frequencies if supplied)"*: `quants_mat.mtx`
+>    - *"For use with --mtx-matrix: force interpretation of matrix to assume cells are by row, rather than by column (default)"*: `Yes`
+>    - *"Label to place in plot title"*: `Barcode rank plot (Alevin-processed)`
+>    - *"Number of bins used in barcode count frequency distribution"*: `50`
+>    - *"Above-baseline multiplier to calculate roryk threshold"*: `1.5`
+>
+> 2. Rename {% icon galaxy-pencil %} the image output `Barcode Plot - Alevin processed barcodes`
+{: .hands_on}
+
+![raw droplet barcode plots - 400k](../../images/wab-alevin-barcodes-400k.png "400k subsample - Alevin processed")
+
+And the full sample looks like:
+
+![raw droplet barcode plots - total](../../images/wab-alevin-barcodes-total.png "Total sample - 32,579,453 reads - Alevin processed")
+
+And to round this off, here's the lung atlas plot.
+
+![raw droplet barcode plots - total](../../images/wab-alevin-barcodes-lung.png "Pretty data - Alevin processed")
```

Comment:
You don't describe this as lung atlas when you mention it above- maybe describe them the same way?

Reply:
Excellent catch! I've added in the lung atlas description